homeassistant/components/media_player/russound_rnet.py

```diff
@@ -85,23 +85,30 @@ def __init__(self, hass, russ, sources, zone_id, extra):
 
     def update(self):
         """Retrieve latest state."""
-        if self._russ.get_power('1', self._zone_id) == 0:
-            self._state = STATE_OFF
-        else:
-            self._state = STATE_ON
-
-        self._volume = self._russ.get_volume('1', self._zone_id) / 100.0
-
+        # Updated this function to make a single call to get_zone_info, so that
+        # with a single call we can get On/Off, Volume and Source, reducing the
+        # amount of traffic and speeding up the update process.
+        ret = self._russ.get_zone_info('1', self._zone_id, 4)
+        _LOGGER.debug("ret= %s", ret)
+        if ret is not None:
```

Comment:
**Preference:**
Our preferred style is the pattern shown below. This saves us from having to indent the rest of the function. This will also line up the indentation of the comments below with the code.
```python
if ret is None:
    _LOGGER.error("Could not update status for zone %s", self._zone_id)
    return

_LOGGER.debug("Updating status for zone %s", self._zone_id)
…
```

Reply:
Hi @armills , thank you for your feedback. Could it be possible to do the merge like this just for this moment, so that I can concentrate on additional enhancements to the Russound RNET component? Right now this component cannot reconnect if there is a communications error (forcing you to a full Hass restart), and would like to make fixing this my first priority. I will have the update ready in a week from now (including this style change). Do you agree?

Reply:
Yep, that's no problem. Not a blocker to merge. Thanks for the contribution! 🎉